components/DBTableMocker.php

```diff
@@ -8,6 +8,20 @@
      * Class which is used for mocking Db Tables which allows proper testing
      */
     class DBTableMocker {
+
+        /**
+         * Name of the column which is being changed
+         *
+         * @var string
+         */
+        private $currentColumn = "";
```

Comment:
Added a field to store the column which is being changed. Might later be turned into an array from string.